packages/help-center/src/styles.scss

```diff
@@ -275,7 +276,8 @@ button.button.back-button__help-center.is-borderless {
 	}
 }
 
-.is-section-checkout {
+body[class*='is-section-'],
+body[class^='is-section-'] {
```

Comment:
Minor thing but I think it's still worth thinking about...
Is it possible to change this targeting for the body?
We create a dependency on how the classes are handled in calypso.
Unwrapping this from the top selector seems to work as well for calypso and the ETK or I'm missing something?

Reply:
Tbh I followed the checkout example, so it may be not necessary to add the check in the css but just use it.